portal-api/api-tests/wipeDB.js

```diff
@@ -19,7 +19,7 @@ const wipe = async (collections) => {
 };
 
 const wipeAll = async () => {
-  const wiped = await wipe(['deals', 'facilities', 'banks', 'transactions', 'industrySectors', 'mandatoryCriteria', 'eligibilityCriteria', 'users']);
+  const wiped = await wipe(['deals', 'facilities', 'banks', 'transactions', 'industrySectors', 'mandatoryCriteria', 'mandatoryCriteriaVersioned', 'eligibilityCriteria', 'users']);
```

Comment:
For clarity, can we prefix gef- in front of the names of gef specific collections in the db. Otherwise, all looks good.

Reply:
database collection prefixed with '-gef' as instructed